files/image_config/monit/proc_stat

```diff
@@ -0,0 +1,122 @@
+#
+  set daemon 60            # check services at 1-minute intervals
+#   with start delay 240    # optional: delay the first check by 4-minutes (by
+#                           # default Monit check immediately after Monit start)
+set logfile syslog
+set idfile /var/lib/monit/id
+#
+###############################################################################
+## Includes all the critical processes running in SONiC containers.
+###############################################################################
+
+# For syncd container.
+check process syncd matching "/usr/bin/syncd --diag"
+	if does not exist then alert
+
+check process dsserve matching "/usr/bin/dsserve"
+	if does not exist then alert
+
+# For SNMP container.
+check process snmpd matching "/usr/sbin/snmpd"
+	if does not exist then alert
+
+check process snmpd_subagent matching "python3.6 -m sonic_ax_impl"
+	if does not exist then alert
+
+# For Teamd container.
+check process teamd matching "/usr/bin/teamd -r -t"
```

Comment:
SONiC can spawn multiple instances of teamd. From our discussion offline, this will only generate a log message if _all_ of the instances stop running. It will be far more beneficial to get notified if _any_ of these processes stop running.